pipelines.py

```diff
@@ -92,6 +92,65 @@ def unet_multitask(config, train_mode):
                   cache_dirpath=config.env.cache_dirpath)
     return output
 
+def two_unets(config, train_mode):
```

Comment:
this name is not very descriptive. Maybe something like `two_unet_specialists` or `unet_mask_contour_specialists` would be better.

Reply:
I agree with @kdziedzic66, that good name for it is: `two_unet_specialists`
@jakubczakon